### ISSUE TITLE
Issue evaluating mobilenet trained from slim/nets

### ISSUE THREAD

**Issue by wassimea** [tensorflow/models#6094]
### System information
- **What is the top-level directory of the model you are using**: slim/nets
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.9
- **Bazel version (if compiling from source)**: NA
- **CUDA/cuDNN version**:  9.0 / 7.4.1
- **GPU model and memory**: GTX 980
- **Exact command to reproduce**:

### Describe the problem
I have trained mobilenet v2 from tensorflow slim/nets for a binary classification problem.

During training, I get decent validation accuracies (96%), but when I try to restore a saved checkpoint to run it on the same validation set, mobilenet returns the exact same probabilities regardless of the input image.

I believe it might be a problem with the model scope.

I use this exact code to restore other models that I have defined and they work without any problems. The problem is only when I try to restore a model trained from research/slim/nets where I get the exact same probabilities regardless of the input image.

### Source code / logs
I use this code to restore the trained mobilenet model

class Detector1(object):
def __init__(self, index,config):#, data_size, batch_size, model_path):

    model_path = '/logs/'
    graph = tf.Graph()
    with graph.as_default():
        self.image_op = tf.placeholder(tf.float32, shape=[1, 48, 48, 3], name='input_image')
        with tf.contrib.slim.arg_scope(mobilenet_v2.training_scope(is_training=False)):
              self.cls_prob, endpoints = mobilenet_v2.mobilenet(self.image_op)
        self.sess = tf.Session(
            config=tf.ConfigProto(allow_soft_placement=True, gpu_options=tf.GPUOptions(allow_growth=True)))
        saver = tf.train.Saver()
        saver.restore(self.sess, '/logs/-600')
def predict(self,image):
    cls_prob = self.sess.run([self.cls_prob], feed_dict={self.image_op: image})
    return cls_prob

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
